src/main/java/com/sandogh/sandogh/base/entity/BaseEntity.java

```diff
@@ -11,13 +11,12 @@
  **/
 @MappedSuperclass
 @Data
+
 public class BaseEntity {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(columnDefinition = "serial")
     private Long id;
 
-    @CreationTimestamp
```

Comment:
Why did you remove this property?
Put it back if removed with no specific intention.

Reply:
I returned the information and we save the time when the user saves